scripts/release/branch_diff.py

```diff
@@ -11,18 +15,66 @@ def __init__(self, remote, name):
 
     @classmethod
     def parse(cls, raw):
+        raw = raw.strip()
         if " -> " in raw:
             return None
         else:
-            parts = raw.split('/')
-            return Branch(parts[0], parts[1])
+            split_point = raw.find('/')
+            if split_point:
+                return Branch(raw[:split_point], raw[(split_point + 1):])
+            else:
+                raise Exception("Unable to parse branch '{}'".format(raw))
```

Comment:
Turns out we had a branch name in the montagu-webapps with a `/` in its name, so this needed adapting.

Reply:
Does this mean that we will choke on any branchname that is not correctly specified (thinking in particular of the xxx-patch-1 branches that direct editing on github generate - like Kim might end up doing)

Reply:
What we are doing here is splitting a remote branch's name. So this is taking something in the form `origin/my_branch` and making sure we get the `my_branch` bit. I think all remote branches will follow this format?